server/src/main/java/com/objectcomputing/checkins/services/memberprofile/MemberProfile.java

```diff
@@ -48,6 +49,7 @@ public MemberProfile() {
     
     @Column(name="pdlId")
     @TypeDef(type=DataType.STRING)
+    @Nullable
```

Comment:
Added nullable to pdlID column since an employee might not have a PDL at all times. Also, not all employees have an assigned PDL.